app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -189,44 +189,6 @@ class KSApolloClient(val service: ApolloClient) : ApolloClientType {
         }
     }
 
-    private fun createCommentObject(commentFr: fragment.Comment?): Comment {
-
-        val badges: List<String>? = commentFr?.authorBadges()?.map { badge ->
-            badge?.rawValue() ?: ""
-        }
-
-        val author = User.builder()
-            .id(decodeRelayId(commentFr?.author()?.fragments()?.user()?.id()) ?: -1)
-            .name(commentFr?.author()?.fragments()?.user()?.name())
-            .avatar(
-                Avatar.builder()
-                    .medium(commentFr?.author()?.fragments()?.user()?.imageUrl())
-                    .build()
-            )
-            .build()
-
-        return Comment.builder()
-            .id(decodeRelayId(commentFr?.id()) ?: -1)
-            .author(author)
-            .repliesCount(commentFr?.replies()?.totalCount() ?: 0)
-            .body(commentFr?.body())
-            .authorBadges(badges)
-            .cursor("")
-            .createdAt(commentFr?.createdAt())
-            .deleted(commentFr?.deleted())
-            .parentId(decodeRelayId(commentFr?.parentId()) ?: -1)
-            .build()
-    }
-
-    private fun createPageInfoObject(pageFr: fragment.PageInfo?): PageInfoEnvelope {
```

Comment:
It has been moved outside the class as a private package method nothing more, I was perfectly able to use it for replies as well 👍